TaskManager_ConsoleApp_Terry/TodoManager/TodoManager.cs

```diff
@@ -38,7 +38,15 @@ public void BulkCreateTodo(string[] Titles)
         }
         public TodoItem DeleteTodo(int Id)
```

Comment:
If you are going to return null, then you need to specify that in the method signature as well. This tells the colling code to also expect a null value. If the calling code gets a null and does not handle it properly, then there is the potential for catastrophic failure.

Might as well change this to a try delete rather than delete as the method 'tries' to delete, if not found, then do nothing.

```suggestion
        public TodoItem? TryDeleteTodo(int Id)
```